nikl_m.py

```diff
@@ -4,6 +4,7 @@
 import os
 import audio
 import re
+from hparams import hparams
```

Comment:
nits: this is not necessary because #74 merged

Reply:
will change it! thanks!